test/transmitter.test.js

```diff
@@ -1,22 +1,42 @@
 const test = require('tap').test;
 const proxyquire =  require('proxyquire');
 const nock = require('nock');
+const needle = require('needle');
 
 const sinon = require('sinon');
 const spy = sinon.spy();
 const debugMock = (loggerType) => (msg) => {spy(msg);};
 const state = require('../lib/state');
+const config = require('../lib/config');
+config.initConfig({projectId: 'some-project-id'});
 const transmitter = proxyquire('../lib/transmitter', {'debug': debugMock});
 
 test('Transmitter transmits 0 events for no events', async function (t) {
   nock('http://host')
-  .post('/method')
-  .reply(200, {});
+    .post('/method')
```

Comment:
again - just making sure it is supposed to be duplicated

Reply:
yes, this tells `nock` to expect two requests (see our e2e test for more reference)